static/app/components/feedback/table/feedbackTable.tsx

```diff
@@ -124,15 +136,19 @@ export default function FeedbackTable({isError, isLoading, data, location}: Prop
 
 function FeedbackDetailsLink({
   organization,
+  project,
   value,
 }: {
   organization: Organization;
+  project: Project;
   value: string;
 }) {
   return (
     <Link
       to={{
-        pathname: normalizeUrl(`/organizations/${organization.slug}/feedback/${value}/`),
+        pathname: normalizeUrl(
+          `/organizations/${organization.slug}/feedback/${project.slug}:${value}/`
```

Comment:
Is this the pattern we've moved away from in replay?

Reply:
ya, we moved away from it. but the feedback read api's need a project name, so i brought it all back